lib/iris/common/metadata.py

```diff
@@ -1462,6 +1577,7 @@ def values(self):
     CoordMetadata.combine,
     CubeMetadata.combine,
     DimCoordMetadata.combine,
+    ConnectivityMetadata.combine,
```

Comment:
Same here... you know what I mean :stuck_out_tongue_winking_eye:

Reply:
c2ac0fc05949f07cd3483e802994708aa427da22